fastestimator/dataset/op_dataset.py

```diff
@@ -65,6 +67,9 @@ def __getitem__(self, index: int) -> Mapping[str, Any]:
             items = {key: np.array([item[key] for item in items]) for key in items[0]}
         else:
             forward_numpyop(self.ops, items, {'mode': self.mode})
+        if self.output_keys:
+            for key in set(items.keys()) - self.output_keys:
```

Comment:
don't need to cast items.keys() to a set for this

Reply:
nice, fixed